Makefile

```diff
@@ -67,7 +67,10 @@ test-short:
 	@go test -mod=readonly -v -run TestOsmoToGaiaRelaying ./_test/
 
 ibctest:
-	cd ibctest && go test -race -v .
+	cd ibctest && go test -race -v -run TestRelayer .
```

Comment:
Looks like this would match both of the test functions -- prefer to match the whole name, taking care to match the updated name.

Reply:
updated